### ISSUE TITLE
Inactive Users Report or API

### ISSUE THREAD

**Issue by lox** [buildkite/feedback#292]
It would be great to have an AP or a report that allowed Organization Admins to determine the last time a user had interacted with the Organization so that inactive accounts could be disabled.

At present this is difficult to expose because we don't have an API for user metadata that relates to an Organization. Users can be part of multiple organizations, so simply exposing a `last_login` on a user would leak private / cross-organization information.

**Comment by KevinGrandon** [comment]
IMO slack has a pretty good model for this. They will automatically detect inactive users (no activity within two weeks), and deactivate their accounts, meaning you won't be billed for them. Perhaps a similar model, where no activity, logins, or API events from a user within a month means they get deactivated.

https://get.slack.help/hc/en-us/articles/218915077#how-inactive-and-deactivated-accounts-affect-billing

**Comment by lox** [comment]
Yup, absolutely. We've discussed that model a lot and philosophically all agree with it @KevinGrandon. This would be the first step towards that model.
